src/test/java/it/gov/pagopa/payhub/auth/mapper/ClientDTO2UserInfoMapperTest.java

```diff
@@ -0,0 +1,56 @@
+package it.gov.pagopa.payhub.auth.mapper;
+
+import it.gov.pagopa.payhub.auth.dto.IamUserInfoDTO;
+import it.gov.pagopa.payhub.auth.dto.IamUserOrganizationRolesDTO;
+import it.gov.pagopa.payhub.auth.utils.Constants;
+import it.gov.pagopa.payhub.model.generated.ClientDTO;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Collections;
+import java.util.UUID;
+
+@ExtendWith(MockitoExtension.class)
+class ClientDTO2UserInfoMapperTest {
+
+	@InjectMocks
+	private ClientDTO2UserInfoMapper mapper;
+
+	@Test
+	void givenDTOWhenApplyTheOk() {
+		// Given
+		String plainClientSecret = UUID.randomUUID().toString();
+		String organizationIpaCode = "organizationIpaCode";
+		String clientName = "clientName";
+		String clientId = organizationIpaCode + clientName;
+
+		ClientDTO clientDTO = ClientDTO.builder()
+			.clientId(clientId)
+			.clientName(clientName)
+			.organizationIpaCode(organizationIpaCode)
+			.clientSecret(plainClientSecret)
+			.build();
+
+		// When
+		IamUserInfoDTO result = mapper.apply(clientDTO);
+		//Then
+		Assertions.assertEquals(
```

Comment:
let's also assert that all result fields are not null (there is an utility method inside TestUtils).
This way, if someone adds a new field and forgets to map it, this test will notify him.

Reply:
I'm going to import the missing class from mypay project